### PR TITLE
Fixed code style for better readability

### PR DESCRIPTION
**Pull request by alexraputa**
| Q                | A
| ---------------- | ---
| Is bugfix?       | no
| New feature?     | no
| Breaks BC?       | no
| Tests pass?      | yes

### CODE REVIEW

**Comment by SilverFire** [comment]
What fixer have you used for this change

**Comment by alexraputa** [comment]
@SilverFire, I used `php-cs-fixer` and hands for some rules :)

**Comment by SilverFire** [comment]
Could you post a fixer config you've applied?

**Comment by alexraputa** [comment]
Do another pull request for it?

**Comment by SilverFire** [comment]
Yes, because we already have one: https://github.com/yiisoft/yii2/blob/master/cs/src/YiiConfig.php

File: tests/framework/helpers/FileHelperTest.php [hunk 1]
```diff
@@ -4,7 +4,6 @@
  * @copyright Copyright (c) 2008 Yii Software LLC
  * @license http://www.yiiframework.com/license/
  */
-
 use yii\helpers\FileHelper;
```
Comment (rob006) [inline tests/framework/helpers/FileHelperTest.php]: Why remove this line?

**Review by rob006 — CHANGES_REQUESTED** [review]
Changes in requirement checker should be reverted - it should work with really old PHP versions, it is better to not touch it at all. :P

File: tests/framework/helpers/FileHelperTest.php (hunk 1 above)
Comment (alexraputa) [reply tests/framework/helpers/FileHelperTest.php]: Maybe after updating to the latest version of the `php-cs-fixer`.

**Pull request closed without merging**